src/Core/Settings.cs

```diff
@@ -273,6 +273,9 @@ public class FileFormatData : AutoConfiguration
         [ConfigComment("If true, folders will be discarded from starred image paths.")]
         public bool StarNoFolders = false;
 
+        [ConfigComment("Whether to automatically use the base model type as part of the model path when downloading using the 'Model Download' tool")]
```

Comment:
3x minor doc nits:
- `.` at end of the sentence
- also it's the `Model Downloader utility`
- Also doc here should ideally be a bit more clean what it does - something like `\nFor example, SDXL models will be placed automatically into an `SDXL/` directory.

Reply:
On it